src/mdbExtensionController.ts

```diff
@@ -48,28 +48,20 @@ export default class MDBExtensionController implements vscode.Disposable {
   _languageServerController: LanguageServerController;
   _webviewController: WebviewController;
 
-  constructor(
-    context: vscode.ExtensionContext,
-    connectionController?: ConnectionController
-  ) {
+  constructor(context: vscode.ExtensionContext, needTelemetry?: boolean) {
```

Comment:
Added returning type for the `openResultAsVirtualDocument` function.

I was actually choosing between `needTelemetry` as a prop and `{needTelemetry: true}` options object :) Since you also like it more, then let's use the object instead!